package.json

```
@@ -24,9 +26,9 @@
 	"homepage": "https://github.com/derhuerst/db-hafas-stations",
 	"repository": "derhuerst/db-hafas-stations",
 	"bugs": "https://github.com/derhuerst/db-hafas-stations/issues",
-	"license": "ISC",
+	"license": "SEE LICENSE IN license-data.md",
```

`license`'s value must be an [SPDX license expression](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/).

I *think* that, when using `db-hafas-stations`, *both* the source code's license as well as the data's license apply.

```suggestion
	"license": "ISC AND CC-BY-4.0 AND ODbL-1.0",
```

---

While definitely more machine-readable, it lacks the rights owner info for CC-BY and ODbL (I had considered adding DBInfraGO as contributor, but that also felt odd). According to [npm docs](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#license), `SEE LICENSE IN` is also valid :D But yeah your suggestion also works for me :)

---

Didn't know about the `SEE LICENSE IN` part. Also, the SPDX expression `LicenseRef-…` would work.

---

> While definitely more machine-readable, it lacks the rights owner info for CC-BY and ODbL

AFAICT that is not the goal behind SPDX License Expressions. I think the idea is only to inform consumers which licenses they'll have to follow, not which authors to credit.